conformance_tests/core/test_memory/src/test_virtual_memory.cpp

```diff
@@ -134,18 +135,30 @@ TEST_F(zeVirtualMemoryTests,
        GivenPageAlignedSizeThenVirtualAndPhysicalMemoryReservedSuccessfully) {
   lzt::query_page_size(context, device, allocationSize, &pageSize);
   allocationSize = lzt::create_page_aligned_size(allocationSize, pageSize);
-  lzt::physical_memory_allocation(context, device, allocationSize,
-                                  &reservedPhysicalMemory);
-  lzt::physical_memory_destroy(context, reservedPhysicalMemory);
+  lzt::physical_device_memory_allocation(context, device, allocationSize,
+                                         &reservedPhysicalDeviceMemory);
+  lzt::physical_memory_destroy(context, reservedPhysicalDeviceMemory);
+}
+
+TEST_F(
+    zeVirtualMemoryTests,
+    GivenPageAlignedSizeThenVirtualAndPhysicalHostMemoryReservedSuccessfully) {
+  lzt::query_page_size(context, device, allocationSize, &pageSize);
+  allocationSize = lzt::create_page_aligned_size(allocationSize, pageSize);
+  lzt::physical_host_memory_allocation(context, allocationSize,
+                                       &reservedPhysicalHostMemory);
```

Comment:
To validate this is actually working, between allocation and destroy, shouldn't we be attempting to at least touch a single of memory to verify it won't segfault?  At least on host host side?

Reply:
@rwmcguir Yes, I plan to add more tests in future PRs. This one mostly just modifies the test harness to support both host & device memory, so we don't have huge PRs that are difficult to understand.